frontend/src/components/AdminContent/index.js

```diff
@@ -0,0 +1,61 @@
+import React, { useState } from 'react'
+import { AdminContentContainer, ContentHeader, ToggleButtonContainer, OrgInfo, OrgInfoImage, OrgInfoName, ContentBody } from './adminContent.styled'
+import { ToggleButton, ToggleButtonGroup } from '@mui/material';
+import ClearIcon from '@mui/icons-material/Clear';
+
+import { orgContext } from '../../pages/admin';
+
+// FIXME: content not aligned with sidebar, need to change sidebar :(
+
+const AdminContent = ({ id, icon, orgName }) => {
+  const [windowSelected, setWindowSelected] = useState("campaigns");
+  const {orgSelected, setOrgSelected, orgList, setOrgList} = React.useContext(orgContext);
```

Comment:
```suggestion
  const {orgSelected, setOrgSelected, orgList, setOrgList} = useContext(orgContext);
```

use imports if we can :)